src/styles/main.scss

```
@@ -0,0 +1,102 @@
+$clock-size: 500px;
+$min-arrow-height: $clock-size*0.3;
+$sec-arrow-height: ($clock-size/2)-10px;
```

```suggestion
$sec-arrow-height: ($clock-size/2) - 10px;
```
check all calculation